src/components/Identity/injectAddQueryStringIdentityToPayload.js

```diff
@@ -25,10 +25,17 @@ export default ({ locationSearch, dateProvider, orgId, logger }) => payload => {
   }
 
   const parsedQueryString = queryString.parse(locationSearch);
-  const queryStringValue = parsedQueryString[queryStringIdentityParam];
+  let queryStringValue = parsedQueryString[queryStringIdentityParam];
+
   if (queryStringValue === undefined) {
     return;
   }
+  if (Array.isArray(queryStringValue)) {
```

Comment:
Should we add a `length` check as well?

Reply:
Yes.
0 instances - undefined
1 instance - string
2 or more - array